tests/src/test/java/me/snowdrop/istio/dt/OpenshiftIT.java

```diff
@@ -0,0 +1,134 @@
+package me.snowdrop.istio.dt;
+
+
+import io.fabric8.openshift.api.model.v3_1.Route;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import me.snowdrop.istio.api.model.IstioResource;
+import org.arquillian.cube.istio.impl.IstioAssistant;
+import org.arquillian.cube.openshift.impl.client.OpenShiftAssistant;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+
+@RunWith(Arquillian.class)
+public class OpenshiftIT {
+    @ArquillianResource
+    private IstioAssistant istioAssistant;
+
+    @ArquillianResource
+    private OpenShiftAssistant openShiftAssistant;
+
+    private static String istioURL;
+    private static String jaegerURL;
+    private static boolean isInit = false;
+    private static List<IstioResource> routeRule = null;
+
+    private final String appUrl = "greeting/";
+
+    @Before
+    public void init() throws Exception {
+        // initiate only before first test
+        if (!isInit) {
+            istioURL = "http://" + getIstioRoute("istio-ingress");
+            jaegerURL = "https://" + getIstioRoute("jaeger-query") + "api/traces";
+
+            routeRule = deployRouteRule("route-rule-redir.yml");
+            await().
+                    pollInterval(1, TimeUnit.SECONDS)
+                    .atMost(1, TimeUnit.MINUTES)
+                    .until(() -> {
+                try {
+                    Response response = RestAssured.get(istioURL + appUrl);
+                    return response.getStatusCode() == 200;
+                } catch (Exception ignored) {
+                    return false;
+                }
+            });
+
+            // jaeger is using HTTPS with self-signed certificate
+            // allow acceptance of non-trusty certificates
+            RestAssured.useRelaxedHTTPSValidation();
```

Comment:
I would prefer we don't do that here, but instead add it to `getJaegerTraces`

Reply:
OK I will move it